saleor/graphql/attribute/mutations/attribute_value_create.py

```diff
@@ -34,10 +41,16 @@ class Arguments:
         )
 
     class Meta:
+        auto_permission_message = False
         model = models.AttributeValue
         object_type = AttributeValue
-        description = "Creates a value for an attribute."
-        permissions = (ProductPermissions.MANAGE_PRODUCTS,)
+        description = (
+            "Creates a value for an attribute.\n\n"
+            "Depending on the attribute type, it requires different permissions to create:\n"
+            "`PRODUCT_TYPE` requires `MANAGE_PRODUCTS` permissions,\n"
+            "`PAGE_TYPE` requires `MANAGE_PRODUCTS` or `MANAGE_PAGES` permissions.\n"
+            "DEPRECATED: it will be changed in 3.15."
```

Comment:
What will be changed in 3.15? It is not fully clear. Do you want to change the required permissions for mutation? It looks like breaking change.

Reply:
In 3.15 for mutations:
 - `AttributeValueCreate`
 - `AttributeValueUpdate`
 - `AttributeValueDelete`
we will require:
 - `MANAGE_PRODUCTS` or `MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES` when dealing with product type attribute,
 - `MANAGE_PAGE` when dealing with page type attribute.

Reply:
- It is not clear from the description, what part of permission is deprecated. It should be included there
- It is a breaking change, even if we fix incorrect permissions (bug), something that was working in 3.14 will stop working in 3.15. 
IMHO  to keep the consistency with our approach for breaking changes in API , this should be depreceated and remove in 4.0. Only preview-feature API can be changed across minor versions.